### DISCUSSION TITLE
Can't renew my student discount

### DISCUSSION THREAD

**Discussion by nema-oss** [community/community#47343] [GitHub Education]
Hey, so my github student discount is about to expire and when I try to renew it, it asks me for my location. At the moment, I am an international student who is away from his home university, for the next 6 months. The email I received states:

   " The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote."
Not my case, even though I do follow my courses from remote.

    "You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so."
I took a picture of the confirmation letter that I got for my mobility. It didn't even care apparently. Plus, it's a document that is several pages long, I'm not sure why the application wants me to take a picture with my camera, wouldn't an upload button be better? 
Is there any way that I can get my discount without using a VPN? This seems like something the developers didn't think completely through.

**Comment by jge162** [comment]
Did you already open a support ticket? We can only discuss it here, we cannot actually renew it. I suggest you do a support -> [ticket](https://support.github.com/contact/education) if you have not already.
